src/main/java/com/google/cloud/tools/managedcloudsdk/components/SdkComponentInstaller.java

```diff
@@ -65,27 +66,29 @@ public void installComponent(
       environment = pythonCopier.copyPython();
     }
 
+    Path workingDirectory = gcloudPath.getRoot();
     List<String> command =
-        Arrays.asList(gcloud.toString(), "components", "install", component.toString(), "--quiet");
-    commandRunner.run(command, null, environment, consoleListener);
+        Arrays.asList(
+            gcloudPath.toString(), "components", "install", component.toString(), "--quiet");
+    commandRunner.run(command, workingDirectory, environment, consoleListener);
```

Comment:
should we make the workingDirectory argument in commandRunner.run not nullable? (It is now.)

Reply:
`CommandRunner` is very generic and can run any process with arbitrary arguments, so I think it's fine to accept null for `workingDirectory`.